apidef/oas/default.go

```diff
@@ -126,6 +149,65 @@ func (as *AuthSources) Import(in string) {
 	}
 }
 
+func (s *OAS) configureMiddlewareOnAllPaths(enabled bool, middleware string) {
+	for path, pathItem := range s.Paths {
+		for _, method := range allowedMethods {
+			if operation := pathItem.GetOperation(method); operation != nil {
+				if middleware == MiddlewareAllowList {
+					s.configureAllowList(enabled, method, path)
+				} else if middleware == MiddlewareValidateRequest {
+					reqBody := operation.RequestBody
+					if reqBody == nil {
+						continue
+					}
+
+					reqBodyVal := reqBody.Value
+					if reqBodyVal == nil {
+						continue
+					}
+
+					media := reqBodyVal.Content.Get("application/json")
+					if media == nil {
+						continue
+					}
+
+					s.configureValidateRequest(enabled, method, path)
+				}
+			}
+		}
+
+	}
+
+}
+
+func (s *OAS) configureAllowList(enabled bool, method, path string) {
```

Comment:
configure or import? in authentication I have used `import` function by creating an interface. Maybe here we can try same. Instead of `*OAS` struct we can call their `Import` functions at once?